dotnet/README.md

```diff
@@ -0,0 +1,44 @@
+# BBS Signatures for .NET Core
+
+[![Build Status](https://dev.azure.com/streetcred/Streetcred/_apis/build/status/streetcred-id.bbs-signatures-dotnet?branchName=master)](https://dev.azure.com/streetcred/Streetcred/_build/latest?definitionId=65&branchName=master)
+
+This is a .NET wrapper for the C callable BBS+ Signatures package located https://github.com/mikelodder7/ffi-bbs-signatures
+The are pre-built dynamic libraries avaliable for each platform in the `libs/` folder. Follow the instructions below if you'd like to build the dependency youself.
```

Comment:
This is a good process for now but now these are colocated how do you feel about us adding an MSBuild pre-build task to this library to automatically build the BBS crate

Reply:
Need to update the url here too.

Reply:
@tplooker The main project requires the presence of all native libs, it would make building the solution too long, and sometimes impossible, for example windows or linux users can't build for all .net targets. We could add a script that downloads the latests libs if not present from github release, once that's up.